go-controller/.golangci.yml

```diff
@@ -15,6 +15,7 @@ linters:
     - goimports
     - gosimple
     - govet
+    - importas
```

Comment:
could you please add some description as comment for why you are adding each import for us? I am OK if you push back to doing so because we can easily google to see what a specific linter is doing.. but each lint seems to be doing many things and we have settings for some of them to be fine grained so would be nice I guess to have some kind of TODO for ppl writing code? maybe we can expand on what we have: https://ovn-kubernetes.io/governance/CONTRIBUTING/#pull-request-checklist  before pushing PRs... -> I mean specifically in terms of the new importas rules we are adding since we want everyone in community to follow these

Reply:
I am confused on what you are asking. The linter runs automatically on each export and errors itself should give sufficient guidance.